arrow-site/docs/docs/quickstart/libraries/README.md

```diff
@@ -102,22 +101,6 @@ Dependency: `arrow-fx-coroutines`
 
 These libraries are hosted inside the Arrow repository building on the core, to provide higher level constructs to deal with concepts rather than code abstraction.
 
-### arrow-syntax
-
-```groovy
-dependencies {
-    implementation "io.arrow-kt:arrow-syntax:$arrow_version"
-}
-```
-
-Multiple extensions functions to work better with function objects and collections.
-
-For function objects, the library provides composition, currying, partial application, memoization, pipe operator, complement for predicates, and several more helpers.
-
-For collections, arrow-syntax provides `firstOption`, tail, basic list traversal, and tuple addition.
```

Comment:
shouldn't this information be included somewhere else instead?

Reply:
It should show up in the List menu automatically as `List` extensions if we move these with the rest of extensions to the `arrow.core` namespace. I personally think we should get rid of the name `syntax` all together as this came from cats to differentiate type class members from actual extensions but in Kotlin it's just extensions and these should be IMO in the same package as the rest.